### PR TITLE
feat(adapters): Add Hyperlane

### PR DESCRIPTION
**Pull request by tmsrjs**

### CODE REVIEW

**Comment by vrtnd** [comment]
please fix typescript errors
`npm run ts`

**Comment by tmsrjs** [comment]
@vrtnd done 👌

**Comment by cmcewen** [comment]
@vrtnd anything we could do to get a review from you here?

File: src/utils/adapter.ts [hunk 1]
```diff
@@ -117,7 +117,7 @@ export const runAdapterToCurrentBlock = async (
     console.warn(`[WARN] No recorded blocks data for ${bridgeDbName}. Error: ${e.message}`);
   }
 
-  const adapter = adapters[bridgeDbName];
+  const adapter = await adapters[bridgeDbName];
```
Comment (vrtnd) [inline src/utils/adapter.ts:120]: let's add adapter.isAsync field and await only in this case

**Comment by tmsrjs** [comment]
@vrtnd would something like this work?
I'm not particularly happy about the amount of repetition but it can easily be abstracted in something like `async function getAdapter(adapterName)`, but then that would have to be `await`ed as well due to the fact that it may have to build the adapter. What do you think?

**Pull request merged**